web/styles/inbox.css

```diff
@@ -338,7 +343,8 @@
             }
 
             .unread_mention_info {
-                grid-area: unread_mention_info;
+                grid-area: mention-and-unread-count-container;
```

Comment:
This seems to be made up; I find no grid definition anywhere with `mention-and-unread-count-container` declared. The `unread_mention_info` area *is* defined, and I do not at all understand why you are messing with that here.